### PR TITLE
Update readme with how to run

### PR DESCRIPTION
**Pull request by tobiasbexelius**
# Jira Issue: []

## What?
-

## Why?
-

## Optional checklist
- [ ] Unit tests written
- [x] Tested locally
- [x] Codescouted

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,4 +24,7 @@ your local `~/.m2/settings.xml` like this:
 </settings>
 ```
 
-1. Run `./mvnw clean install` on root level
+###To run locally:
+1. Run postgres: `docker start hedvig`
+1. Run redis: `redis-server --daemonize yes`
+1. Run `MemberRegistryApplication` with the profile: `development`
```
Comment (vonElfvin) [inline README.md:30]:
```suggestion
###To run locally:
1. Ensure postgres is running: `docker start <image-id>`
1. Run redis: `hedvig start redis` or `redis-server --daemonize yes`
1. Run `MemberRegistryApplication` with the profile: `development`
```

File: README.md [hunk 2]
```diff
@@ -24,4 +24,7 @@ your local `~/.m2/settings.xml` like this:
 </settings>
 ```
 
-1. Run `./mvnw clean install` on root level
+###To run locally:
+1. Run postgres: `docker start <postgres container name>`
```
Comment (Michael-Duivestein) [inline README.md:28]: Not part of the PR, but here's a handy tip:
Give the postgres container name (like pgdbDev). Create an alias like the following to start it up quickly:
`alias sdb='docker start pgdbDev'`

File: README.md (hunk 1 above)
Comment (tobiasbexelius) [reply README.md]: I made similar changes before I saw your suggestion, have a look 🙂

**Review by vonElfvin — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -24,4 +24,7 @@ your local `~/.m2/settings.xml` like this:
 </settings>
 ```
 
-1. Run `./mvnw clean install` on root level
+###To run locally:
```
Comment (vonElfvin) [inline README.md:27]:
```suggestion
### To run locally:
```
